qt/widgets/common/src/PropertyHandler.cpp

```diff
@@ -187,6 +188,23 @@ class CreateAttributeProperty : public Mantid::API::IFunction::ConstAttributeVis
   QString m_name;
 };
 
+/**
+ * Populate ties on parameter properties of child functions
+ */
+void PropertyHandler::initTies() {
+  for (size_t iparam = 0; iparam < function()->nParams(); iparam++) {
```

Comment:
Can we hoist the `function()` call above?  C++ will execute this call each time for no benefit as it potentially has side-effects:
```c++
auto &func = function();
for (size_t iparam = 0; iparam < func->nParams(); iparam++)
```

Reply:
Thanks, have replaced `function() `calls (and also `m_fun`) with `m_cf`